public/index.html

```diff
@@ -5,10 +5,25 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="OlWebsite for the oneleif open source group"
-    />
+    
+    <meta name="description" content="Grow and create together." />
+
+    <!-- Twitter Card data -->
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@oneleifdev">
+    <meta name="twitter:title" content="oneleif">
+    <meta name="twitter:description" content="Grow and create together.">
+    <!-- Twitter Summary card images must be at least 120x120px -->
+    <meta name="twitter:image" content="https://secure.meetupstatic.com/photos/event/4/a/9/e/600_488179102.jpeg">
+
+    <!-- Open Graph data -->
+    <meta property="og:title" content="oneleif" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://oneleif.com/" />
+    <meta property="og:image" content="https://secure.meetupstatic.com/photos/event/4/a/9/e/600_488179102.jpeg" />
```

Comment:
This image is pointing towards the oneleif image for the meet-up site. Not sure if this will work but trying it out for now till we provide our own images.